css/base.css

```diff
@@ -0,0 +1,51 @@
+:root {
+    /* colors palette*/
+  --background-color: #fff8e7;
+  --accent-color: #8b82fa;
+  --cta-color: #f34e4e;
+  --font-color-dark: #160C28;
+  --font-color-light: #160c28a1;
+
+  /* box shadow */
+  --box-shadow: rgba(0, 0, 0, 0.123) 0px 25px 20px -20px;
+
+   /* hover colors */
+  --hover-color-links: #fbc416;
+  --hover-color-links-light: #fbc6169d;
+  --hover-color-btn-prim: #f54242;
+  --hover-color-btn-sec: #f54242;
+
+   /* font family */
+  --font-family: 'Inter', sans-serif;
+
+   /* sizes */
+  --space-0: 0;
```

Comment:
since you are using this for sizing, maybe you can use a better variable name
like `--size-0` and so on

Reply:
Yes that's sounds right. `space` would be better for spacing.